### PR TITLE
report channel counts and modes in status

### PR DESCRIPTION
**Pull request by matiasw**
See https://github.com/progval/Limnoria/issues/1183

If accepted, could you please add the `hacktoberfest-accepted` label? I understand you do not want your repository to participate, but this PR still can.

### CODE REVIEW

File: plugins/Status/plugin.py [hunk 1]
```diff
@@ -76,15 +76,53 @@ def status(self, irc, msg, args):
...
+                    normal_counts[network_name] += 1
+
+        # Prepare the response
+        response_lines = []
+        for network_name in channel_counts:
+            response_lines.append(
+                f"I am connected to {network_name} as {nicks[network_name]}, Channels: {channel_counts[network_name]}, "
+                f"Ops: {op_counts[network_name]}, Half-Ops: {halfop_counts[network_name]}, "
+                f"Voiced: {voice_counts[network_name]}, Regular: {normal_counts[network_name]} "
+            )
```
Comment (progval) [inline plugins/Status/plugin.py:120]: Please keep the `_()` wrapper and `%s` substitution so it's translatable.

**Review by progval — CHANGES_REQUESTED** [review]
> If accepted, could you please add the `hacktoberfest-accepted` label?

Sure, that's a good contribution

**Comment by matiasw** [comment]
Is this more or less what you meant? If not, what would be the desired format for the output?

**Comment by progval** [comment]
thanks!

**Pull request merged**